mne/decoding/search_light.py

```diff
@@ -319,7 +331,9 @@ def _sl_fit(estimator, X, y, pb, **fit_params):
         est = clone(estimator)
         est.fit(X[..., ii], y, **fit_params)
         estimators_.append(est)
-        pb.update(ii + 1)
+
+        if pb is not None:
```

Comment:
Are there still cases where `pb is None`? If so it seems to imply there is more code to convert

Reply:
Still wondering about this